src/nuget.config

```diff
@@ -8,6 +8,7 @@
   <packageSources>
     <add key="nuget.org" value="https://www.nuget.org/api/v2/" />
     <add key="particular myget.org" value="https://www.myget.org/F/particular/" />
+    <add key="Local Integration Source" value="..\nugets" />
```

Comment:
Is that for local development with unreleased packages?

Reply:
yes. u have an integration solution for smoke testing the full msbuild pipeline